pkg/sql/sem/tree/pretty.go

```diff
@@ -496,6 +496,8 @@ func (node *Limit) docTable(p *PrettyCfg) []pretty.TableRow {
 			e = StripParens(e)
 		}
 		res = append(res, p.row("LIMIT", p.Doc(e)))
+	} else if node.LimitAll {
+		res = append(res, p.row("", pretty.Keyword("LIMIT ALL")))
```

Comment:
I think this would be better because it'll line up the LIMIT clauses:

`res = append(res, p.row("LIMIT", pretty.Keyword("ALL")))`